tidysic/audio_file.py

```diff
@@ -28,81 +28,48 @@ class AudioFile(object):
     Class describing a file, and its associated tags
     '''
 
-    accept_all_guesses = False
-
     def __init__(self, file):
         self.file = file
         self.tags = self.read_tags()
 
-    def guess_tags(self, dry_run):
+    def ask_and_set_tag(self, tag: Tag):     
```

Comment:
This function is doing way too many things on its own, and this is obscuring the code.
Let's break it together:
- Informing the user on how to input a value
- Retrieving the value
- Validating the value
- setting an edge case that can be understood only by reading the logger's string at the top of the function.

I would suggest breaking it into calls to smaller functions. In this way, it tells a better story while bringing a clearer separation of concern. Here is my attempt at refactoring this (not the absolute truth, but more like an example):

```python
def ask_and_set_tag(self, tag): 
'''
Ask the user to provide one of the file's tags, and set it.

Args:
    tag (Tag): Tag the user will be asked to provide

Returns:
    Union[str, int]: Value entered by the user
'''
    self._explain_set_tag(tag)
    value = self._ask_tag(tag)
    self.tags[tag] = value
    return self.tags[tag]

def _explain_set_tag(self, tag: Tag) -> None:
    logger.log(
        f'File [blue]{self.file}[/blue] '
        f'is missing its [green]{tag.name} tag[/green].'
    )
    logger.log(
        'Please provide it now. If the value -1 is given, the file will be '
        f'placed in a [green]Unknown {tag.name}[/green] folder'
    )

def _ask_tag(self, tag: Tag):
    value = input(f'{tag.name}: ')
    value.strip()
    if not AudioFile._validate_input_tag(value):
        logger.log('The inputted value must be a number.')
        return self._ask_tag(tag)
    return value

@staticmethod
def _validate_input_tag(value) -> bool:
    return type(value) == int or value == ''
```